packages/tools/src/synchronizers/callbacks/stackImageSyncCallback.ts

```diff
@@ -69,21 +69,23 @@ export default async function stackImageSyncCallback(
   if (frameOfReferenceUID1 === frameOfReferenceUID2) {
     // if frames of references are the same we can use the absolute
     // imagePositionPatient to find the closest image in the target viewport's stack
-    const closestImageIdIndex = _getClosestImageIdIndex(
-      sourceImagePositionPatient,
-      targetImageIds
-    );
+    if (coPlanarViewports(sViewport, tViewport)) {
```

Comment:
@sedghi - the reason we are adding the changes is that the synchronizer can be added in other situations than the tool, such as being useful in a hanging protocol.  As well, if the tool is active, and one switches to a different set of series, then the coplanar check needs to be in the tool itself.